versioneer.py

```diff
@@ -1,5 +1,5 @@
 
-# Version: 0.10
+# Version: 0.15
```

Comment:
Please confirm that these changes are created by the tool and so I can ignore them in the review.